nitime/analysis/tests/test_granger.py

```diff
@@ -109,5 +109,10 @@ def test_GrangerAnalyzer():
     # Test inputting ij:
     g2 = gc.GrangerAnalyzer(ts1, ij=[(0, 1), (1, 0)])
 
+    # g1 agrees with g2
+    npt.assert_almost_equal(g1.causality_xy[0, 1], g2.causality_xy[0, 1])
```

Comment:
So if we had: 

    g2 = gc.GrangerAnalyzer(ts1, ij=[(1, 0), (0, 1)])

then the reversal would make sense, right?